src/main/java/org/folio/circulation/support/CqlSortDirection.java

```diff
@@ -0,0 +1,16 @@
+package org.folio.circulation.support;
+
+public enum CqlSortDirection {
+
+  ASC("sort.ascending"), DESC("sort.descending");
```

Comment:
Personally, I'd be tempted to use the full words here. Is there a reason to keep them short?

Reply:
Renamed to full words